app/views/users/new.html.erb

```diff
@@ -0,0 +1,65 @@
+<% if user_signed_in? %>
+  <% if user_can_add_new_member(current_user) %>
+	<h1>Create User</h1>
+
+	<%= form_for User.new , :url => {:action => "add_user"} do |form| %>
```

Comment:
Do we have to recreate this form?  We can't use the current form we have?

Reply:
I can use previous one it won't be a problem for me. Do you want me to just redirect sign up page or add some custom text and buttons with it?